dbms/tests/queries/0_stateless/00700_decimal_empty_aggregates.sql

```diff
@@ -0,0 +1,90 @@
+DROP TABLE IF EXISTS decimal;
+
+CREATE TABLE decimal
+(
+    a Decimal32(4),
+    b Decimal64(8),
+    c Decimal128(8)
+) ENGINE = Memory;
+
+SELECT count(a), count(b), count(c) FROM decimal;
+SELECT [min(a), max(a)], [min(b), max(b)], [min(c), max(c)] FROM decimal;
+
+SELECT sum(a), sum(b), sum(c), sumWithOverflow(a), sumWithOverflow(b), sumWithOverflow(c) FROM decimal;
+SELECT sum(a), sum(b), sum(c), sumWithOverflow(a), sumWithOverflow(b), sumWithOverflow(c) FROM decimal WHERE a > 0;
+SELECT sum(a), sum(b), sum(c), sumWithOverflow(a), sumWithOverflow(b), sumWithOverflow(c) FROM decimal WHERE a < 0;
+SELECT sum(a+1), sum(b+1), sum(c+1), sumWithOverflow(a+1), sumWithOverflow(b+1), sumWithOverflow(c+1) FROM decimal;
+SELECT sum(a-1), sum(b-1), sum(c-1), sumWithOverflow(a-1), sumWithOverflow(b-1), sumWithOverflow(c-1) FROM decimal;
+
+SELECT avg(a), avg(b), avg(c) FROM decimal;
```

Comment:
Need also toTypeName(a), toTypeName(b), toTypeName(c) columns.

Reply:
```
SELECT 
    quantiles(0.5, 0.6, 0.99999)(d) AS q, 
    toTypeName(q), 
    avg(d) AS a, 
    toTypeName(a)
FROM test2

┌─q───────────────────┬─toTypeName(quantiles(0.5, 0.6, 0.99999)(d))─┬─────a─┬─toTypeName(avg(d))─┐
│ [3.500,3.800,4.999] │ Array(Decimal(10, 3))                       │ 3.500 │ Decimal(10, 3)     │
└─────────────────────┴─────────────────────────────────────────────┴───────┴────────────────────┘

1 rows in set. Elapsed: 0.017 sec. 

```
result types for avg fixed.
tests updated